app/controllers/notifications/reads_controller.rb

```diff
@@ -3,6 +3,7 @@ class ReadsController < ApplicationController
     def create
       render plain: "" && return unless current_user
 
+      # TODO: notification controller read
```

Comment:
I found `TODO`s on this pr not very informative, so it makes sense to remove them before the review (if you use them while developing).